proxy/proxy.go

```diff
@@ -350,11 +352,26 @@ func initLogging() {
 	}
 }
 
+var enablePprof bool
+
+func setupPprof() {
+	if !enablePprof {
+		return
+	}
+
+	go func() {
+		http.ListenAndServe("localhost:6060", nil)
```

Comment:
Yes agreed, and as you and @jcvenegas suggested, it would be great to have the ip and port as options.